pages/integrations/amazon_eventbridge.md

```diff
@@ -692,3 +699,45 @@ AWS EventBridge has strict limits on the size of the payload as documented in [A
   }
 }
 ```
+
+<h3 id="audit-event-logged">Audit Event Logged <%= pill "PRIVATE BETA", "march-23-release" %> </h3>
+
+[Audit log](/docs/pipelines/audit-log) is only available to Buildkite customers on the [Enterprise](https://buildkite.com/pricing) plan
+
+```json
+{
+    "version": "0",
+    "id": "...",
+    "detail-type": "Audit Event Logged",
+    "source": "...",
+    "account": "...",
+    "time": "2023-03-07T23:14:43Z",
+    "region": "us-east-1",
+    "resources": [],
+    "detail": {
+        "version": 1,
+        "organization": {
+            "uuid": "...",
+            "graphql_id": "...",
+            "slug": "buildkite"
+        },
+        "event": {
+            "uuid": "...",
+            "occurred_at": "2023-03-25 23:14:43 UTC",
+            "type": "ORGANIZATION_UPDATED",
+            "data": {
+                "name": "Buildkite",
```

Comment:
```suggestion
                "name": "Buildkite"
```

Removing the trailing comma makes this valid JSON which you can then test in the AWS test event pattern thingy